### ISSUE TITLE
Strange censor flag

### ISSUE THREAD

**Issue by orian34** [higurashi-mod/watanagashi#77]
https://github.com/07th-mod/watanagashi/blob/abe00721d75fefbfa3efc3822467b4e0b5ac8f94/Update/zwata_009_02_vm00_n01.txt#L43

I noticed this one was quite strange. The dialogue is exactly the same, except the original version says the voice is of a middle aged man. But Kimiyoshi(can't recall who that is. Maybe irrelevant) stills says my old man, which is an odd way of talking for a middle aged man. 
Anyway, it feels like quite an odd change, as because of that, all the voice acting is lost(since it refers to a middle aged man). I mean, if the character is never to be seen again, I guess it doesn't matter...

What are your thoughts on that, Norgus?

Tl;dr : I find the brutal change from a girl to a middle-aged man quite strange, so I'm wondering if it matters, as I don't know if it's an irrelevant side character or not.

PS : I'll make the changes, if there are to be done.

**Comment by orian34** [comment]
Nevermind, I just read the tip after that... and now it's inverted. The old man is the one who made a call, which makes the caller being a girl inconsistent?

**Comment by enumag** [comment]
The old man they're talking about is Kimiyoshi Kiichiro ([this guy](https://vignette.wikia.nocookie.net/whentheycry/images/9/99/Kimi_sifuku.png/revision/latest?cb=20140816170947)), but he is not the one on the phone. The caller is some other member of the Kimiyoshi family. In the original it was some middle-aged man, possibly Kiichiro's son or something. In PS3 they changed the dialogue so that the caller is actually Kimiyoshi Natsumi - a green-haired girl which you'll see later in Someutsushi.

Does it make sense like this? Or is there some inconsistency in the scripts?

**Comment by orian34** [comment]
Yeah, that's kindof what I had pieced together. It's all good then, I'll close it.
